src/test/run-rbd-unit-tests.sh

```diff
@@ -14,4 +14,6 @@ do
     RBD_FEATURES=$i unittest_librbd
 done
 
+unset RBD_FORCE_ALLOW_V1
```

Comment:
Nit: why is this needed?

Reply:
It seems it's not -- I fixed it.
Thanks for your feedback